### ISSUE TITLE
Crash on deleting objects or waypoints

### ISSUE THREAD

**Issue by cameralibre** [synfig/synfig#376]
**Synfig version & platform**: synfigstudio-1.3.3-20170430-linux-64bit.appimage, Ubuntu GNOME 17.04

**Issue description**:

I can open the project. I can add a new shape. But I can't delete that shape, or anything else in the project. There is always a short pause and then the application crashes.

It is a large project (67MB uncompressed, 3000f) as it deals with representations of brain waves at different zoom levels, so some of the vector outlines have hundreds of vertices, and there are dozens of these outlines. 
I have had no indication of what the problem might be or any error messages - the project is attached.
[Synfig_crash.zip](https://github.com/synfig/synfig/files/1061048/Synfig_crash.zip)

Opening other projects with 1.3.3 seems to work. Opening the problematic file with version 1.3.2 appears to work. So using 1.3.2 I will break up the project with into smaller sections, before trying 1.3.3 again.

Synfig has been crashing for me a lot recently, are there debugging/logging tools that I can use to generate more useful information about these crashes?

**Comment by cameralibre** [comment]
nope, same problem with the stable release. Backups of this project also seem to be affected.

**Comment by morevnaproject** [comment]
Whoa, that's a LOT of vertices! ^__^ I have tested your file with 1.3.1 on Linux. I was able to open it. And I was able to delete layer. It hanged when I tried to remove waypoint (the one ,which combines all waypoint from Vertices list). I will keep investigating.

Regarding the debugging tools. At the moment @blackwarthog is thinking how to introduce at least some minimal debugging features integrated into binary build.  He is not sure how to do that yet.

**Comment by FirasH** [comment]
Cannot reproduce with 1.5.0 4f6e09 on openSUSE 15.1

**Comment by Keyikedalube** [comment]
> Regarding the debugging tools. At the moment @blackwarthog is thinking how to introduce at least some minimal debugging features integrated into binary build. He is not sure how to do that yet.

#391

**Comment by Keyikedalube** [comment]
Same here... can't reproduce the crash on latest stable release v1.4.2
